server/package.json

```diff
@@ -64,4 +64,4 @@
   },
   "author": "Sajan Rajdev",
   "license": "ISC"
-}
+}
```

Comment:
Looks like you removed a blank line at the end of this file. So that we can have a cleaner git diff, can you please put it back? This helps for for the current git diff because it won't highlight it like this. And it helps for any future diffs that may add things to the bottom of the file. Thanks!